anitab-org/bridge-in-tech-web
# Feat: Create and Update program page

### Description
Add create and update Program page functionalities

Fixes #53, #78

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
* to Create a program:
- login as a user that has is a representative of an organization and has created an organization
- go to `My Space` > `My Organization` > on the bottom left of the page, click `Go to Programs` > click on `Add Program` button
- fill the details on the Program Profile form page
- click submit
- on successful process, see response below

<img width="815" alt="Screen Shot 2020-08-25 at 12 48 39 pm" src="https://user-images.githubusercontent.com/29667122/91117676-89582f80-e6d2-11ea-859d-df70d4cd1115.png">

* to Update a program: (WIP)
- Now view the program you've just created to make sure it's listed in the programs list (here my new program is Program B)
<img width="974" alt="Screen Shot 2020-08-25 at 1 01 33 pm" src="https://user-images.githubusercontent.com/29667122/91125902-2c667480-e6e6-11ea-84c7-4e0d2a2e908d.png">

- I then updated the program start and date date. On sucessful process, see message below
<img width="986" alt="Screen Shot 2020-08-25 at 1 03 31 pm" src="https://user-images.githubusercontent.com/29667122/91126035-6df71f80-e6e6-11ea-8679-b68f2016286b.png">

- here's the list after the program was updated
<img width="932" alt="Screen Shot 2020-08-25 at 1 05 26 pm" src="https://user-images.githubusercontent.com/29667122/91126103-8ff0a200-e6e6-11ea-8fa8-e49be22fb931.png">



### Checklist:

**Code/Quality Assurance Only**
